### PR TITLE
Test/email servlet utility methods

### PR DESCRIPTION
**Pull request by abundis28**
# What does this PR do?
- Test utility methods.

**Related Issues**
- NA

---
**Merge Checklist**
- [ ] Branch is up to date
- [ ] No code style issues

### CODE REVIEW

File: src/test/java/com/google/sps/UtilityTest.java [hunk 1]
```diff
@@ -115,4 +129,57 @@ public void nullValue() {
 
     Assert.assertEquals(actual, expected);
   }
+
+  @Test
```
Comment (OUMontiel) [inline src/test/java/com/google/sps/UtilityTest.java:133]: nit: Add comment before this test.
`/** Tests for getUsersToNotify() function */`

File: src/test/java/com/google/sps/UtilityTest.java [hunk 2]
```diff
@@ -115,4 +129,57 @@ public void nullValue() {
...
+    String typeOfNotification = "";
+    int modifiedElementId = 2;
+
+    List<Integer> actual = Utility.getUsersToNotify(typeOfNotification, modifiedElementId, request);
+    List<Integer> expected = new ArrayList<>();
+    
+    Assert.assertEquals(actual, expected);
+  }
+
+  @Test
```
Comment (OUMontiel) [inline src/test/java/com/google/sps/UtilityTest.java:159]: nit: Add comment before this test.
`/** Tests for buildComment() function */`

File: src/test/java/com/google/sps/UtilityTest.java (hunk 1 above)
Comment (abundis28) [reply src/test/java/com/google/sps/UtilityTest.java:134]: Done!

File: src/test/java/com/google/sps/UtilityTest.java (hunk 2 above)
Comment (abundis28) [reply src/test/java/com/google/sps/UtilityTest.java:160]: Done!

File: src/test/java/com/google/sps/UtilityTest.java [hunk 3]
```diff
@@ -130,6 +136,31 @@ public void tryParseInt_nullValue_returnsZero() {
     Assert.assertEquals(actual, expected);
   }
 
+  /** Tests for getUsersToNotify(). */
+  @Test
+  public void getUserToNotifyQuestion() {
```
Comment (gloriapark) [inline src/test/java/com/google/sps/UtilityTest.java:141]: A common style practice for naming java unit test functions it to include the method being tested, the test condition, and the expected outcome, so something like: getUsersToNotify_Question_Success() or getUsersToNotify_NoType_ReturnsEmpty()

Doing this makes it really explicitly clear what is being tested, how it's being tested, and what we expect to see. Consider updating your methods to also follow this pattern!

It's not technically part of the public github Java style guide, so I'll leave it up to you - but a good habit to have :)

File: src/test/java/com/google/sps/UtilityTest.java [hunk 4]
```diff
@@ -223,7 +254,6 @@ public void getUsername_zeroId_returnsEmptyString() {
     Assert.assertEquals(actual, expected);
   }
 
-  @Test
```
Comment (gloriapark) [inline src/test/java/com/google/sps/UtilityTest.java:226]: I don't think this was meant to be deleted?

File: src/test/java/com/google/sps/UtilityTest.java (hunk 3 above)
Comment (abundis28) [reply src/test/java/com/google/sps/UtilityTest.java]: Done!

File: src/test/java/com/google/sps/UtilityTest.java (hunk 4 above)
Comment (abundis28) [reply src/test/java/com/google/sps/UtilityTest.java]: Ack!

**Pull request merged**